ramls/instance-storage.raml

```diff
@@ -92,6 +95,36 @@ resourceTypes:
           Instances are stored and accessed by a hash of key properties. The rules which govern
           how instance hashes are computed are business rules and defined in the service layer.
           the storage layer only knows how to insert or retrieve instance records by ID.
+      /preceding-succeeding-titles:
```

Comment:
Why has this endpoint been added to the instance-storage interface when there is a separate preceding / succeeding titles interface?

Reply:
Its because `/instance/<id>/preceding-succeeding-titles` endpoint has direct relation to the instance. It will be incorrect to add this endpoint to the preceding / succeeding titles interface because it operates more narrow resources.

Reply:
> Its because `/instances/<id>/preceding-succeeding-titles` endpoint has direct relation to the instance. 

All [preceding / succeeding titles](https://github.com/folio-org/mod-inventory-storage/blob/16ecdd1b28d55e30ca8b4135eb84080d9adda755/ramls/instanceprecedingsucceedingtitle.json#L11) already have a direct relationship with instances

Could an endpoint be added to the `preceding / succeeding titles` interface that was also specific to an instance instead?

bq. It will be incorrect to add this endpoint to the preceding / succeeding titles interface because it operates more narrow resources.

What do you mean by narrow resources?

At the moment, it is possible to implement the `instance storage interface` without any recognition of preceding / succeeding titles. If this endpoint is added to the instance storage API this would no longer be true, and I think would undermine the intent of `preceding / succeeding titles` being a separate interface.

cc: @nielserik

Reply:
Hey @marc-johnson. I moved it to another interface.

Reply:
@psmagin 

> I moved it to another interface.

Thank you